src/tooltip.tsx

```diff
@@ -28,7 +27,7 @@ const noOp = () => {};
 
 type Position = 'top' | 'bottom' | 'left' | 'right';
 
-const getWidthDesktop = (customWidth?: number) => (customWidth ? customWidth : defaultWidthDesktop);
+const getWidthTooltip = (customWidth?: number) => (customWidth ? customWidth : '');
```

Comment:
If we don't need a default desktop width, then this function doesn't make sense anymore, please just remove it.